index.html

```diff
@@ -1,111 +1,185 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Quiz App -- Quizzy</title>
-    <link rel="stylesheet" href="css/style.css">
-</head>
+    <link rel="stylesheet" href="css/style.css" />
+    <script src="javascript.js" defer></script>
```

Comment:
And I forgot: usually the starting point of a JavaScript file is called either `main.js` or `index.js` or `script.js`. It's another one of these naming conventions developers agreed on 🙂 
So in short: would be great if you could rename your file!